ethereum-network-partition-test/main_test.go

```diff
@@ -111,6 +111,8 @@ func TestNetworkPartitioning(t *testing.T) {
 
 	enclaveCtx, err := kurtosisCtx.CreateEnclave(ctx, enclaveId, isPartitioningEnabled)
 	require.NoError(t, err, "An error occurred creating the enclave")
+	// we only stop the enclave instead of destroying & cleaning it as this allows users  to debug  their enclave after the tests are run
+	// we recommend using `DestroyEnclave` along with `Clean` to clean up enclaves if you don't want remaining artifacts
```

Comment:
@h4ck3rk3y you don't actually need `Clean` - just `DestroyEnclave` is enough to completely remove everything related to an enclave. `Clean`  is just shorthand for "destroy a lot of enclaves"

Reply:
You are right. I used the test suite to confirm the behavior, but seems like the test suite has a bug. Have filed 

https://github.com/kurtosis-tech/kurtosis/issues/647